CONTRIBUTING.md

```diff
@@ -139,7 +139,7 @@ Most changes don't require much "process". If your change is relatively straight
     * [Pull Request](https://github.com/bevyengine/bevy/pulls) (or PR for short): A request to merge code changes. This starts our "review process". You are welcome to start with a pull request, but consider starting with an Issue or Discussion for larger changes (or if you aren't certain about a design). We don't want anyone to waste their time on code that didn't have a chance to be merged! But conversely, sometimes PRs are the most efficient way to propose a change. Just use your own judgement here.
 2. Other community members review and comment in an ad-hoc fashion. Active subject matter experts may be pulled into a thread using `@mentions`. If your PR has been quiet for a while and is ready for review, feel free to leave a message to "bump" the thread, or bring it up on Discord in an appropriate engine development channel.
 3. Once they're content with the pull request (design, code quality, documentation, tests), individual reviewers leave "Approved" reviews.
-4. After consensus has been reached (typically two approvals from the community or one for extremely simple changes) and CI passes, the [S-Ready-For-Final-Review](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AS-Ready-For-Final-Review) label is added.
+4. After consensus has been reached (typically two approval comments from the community or one for extremely simple changes) and CI passes, the [S-Ready-For-Final-Review](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AS-Ready-For-Final-Review) label is added.
```

Comment:
Approvals are not necessarily comments (The comment field can be left empty), and I don't see how this would encourage people to engage.

List item 1 starts with "A community member (that's you!)", and list item 2 explicitly states that community members are the ones reviewing PRs, and it is repeated here as "approvals from the community".
I don't see how this could be made more clear.

Though, maybe the writing could be a little less stiff in places.

Reply:
> As "community members" don't have the rights to "approve", they only have comments as a tool to approve.

They do? I just checked with my professional GitHub account which is not part of Bevy org, and I can approve PRs

Reply:
That's the kind of misunderstanding I wanted to address !

My understanding is that they can approve "implicitly" by leaving a comment stating they approve 🤷.
Or the fact that non-org members cannot approve "officially" was not known by rulemakers.

Reply:
Indeed! Thanks for correcting me 😄